[refs #00104] Move JS variables to separate file

Android63 opened the pull request:
To make them available to framework consumers, move JavaScript
variables from index.html to their own file.

Android63 committed:
[refs #00104] Move JS variables to separate file

To make them available to framework consumers, move JavaScript
variables from index.html to their own file.

cehwitham commented:
Can @csswizardry review this (and the previous changes to JS files) before this gets merged please?

csswizardry commented:
Is this the entirety of the work? There’s still a bunch of JS left in the page, is all.

Android63 committed:
[refs #00104] Move cookies JS to separate file

Put JavaScript for handling cookies ribbon to its own file.

Android63 committed:
[refs #00104] Add JS check

Add check for JavaScript.

Android63 committed:
[refs #00104] Remove javascript-variables file

Since functionality is better described as javascript-check.

Android63 commented:
@csswizardry I've now put all the JS into separate files. Please note I changed one file from javascript-variables.js to javascript-check.js. This was because I *think* I better understand the purpose of that particular script, but I could be wrong.

Android63 committed:
[refs #00104] Use localStorage for cookie ribbon

Store user's decision to close cookie ribbon in browser's local
storage area - and check this before displaying ribbon.
